### DISCUSSION TITLE
my avail points dont show in leaderboard

### DISCUSSION THREAD

**Discussion by oolukman** [availproject/avail#339] [Q&A] [answered]
5H9GtvNWHJRq4DfkgCRiFULRJh5Uxpvd4Pjo1KmyHTwE3wmr

i sent everyday 1 avail to another adresses. first day it showed on leaderboard with 10 points. But after that day, my wallet adress dont show on leaderboard. What is the problem i didnt understand...

**Comment by oolukman** [comment]
![Ekran görüntüsü 2023-12-10 171126](https://github.com/availproject/avail/assets/98649963/55dcd04d-3873-4dcb-92e9-4acd995707bd)

**Comment by Leouarz** [comment, marked as answer]
Hello, seems like you did not get your AVL from the faucet.
Also, it's not really the place to discuss this subject.
